project.md

```diff
@@ -7,62 +7,195 @@ good way to meet their needs.
 
 ### What's the need?
 
-_What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help
-them?_
+Current presentation softwares all have roughly the same output: a deck of
+slides that can be played through linearly.
+For my project, I hope to create a DSL that expands the capabilities of the user
+to make more dynamic presentations that don't have to follow this format.
+Some restrictive elements of traditional presentations I want to change are:
+* All the information on previous slides is lost on the next (unless explicitly
+copied)
+* It is difficult to change styles of components (size, color, transparency,
+etc.) other than visibility at each transition
+* Examining parts of large diagrams typically involves disjointed transitions
+
+This would be more geared toward those familiar with web design since it would
+borrow many of the behaviors and syntax rules from CSS/JS.
+Although this target audience may make the tool seem more restrictive, I believe
+it will offer an avenue for users to more rapidly understand how to use this DSL
+since it won't be a completely foreign language.
```

Comment:
Your description of a presentation kind of sounds like a video, so I see some similarities to my project.